test/integration/case-management.v1.test.js

```diff
@@ -0,0 +1,487 @@
+/* eslint-disable no-console */
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+'use strict';
+
+const CaseManagementV1 = require('../../dist/case-management/v1');
+const authHelper = require('../resources/auth-helper.js');
+
+// testcase timeout value (60s).
+const timeout = 120000;
+
+// Location of our config file.
+const configFile = 'case_management.env';
+
+// Use authHelper to skip tests if our configFile is not available.
+const describe = authHelper.prepareTests(configFile);
+
+// Retrieve the config file as an object.
+// We do this because we're going to directly use the
+// config properties, rather than let the SDK do it for us.
+const config = authHelper.loadConfig();
+
+describe('CaseManagementV1_integration', () => {
+  jest.setTimeout(timeout);
+
+  let service;
+  const testAccountId = config.CASE_MANAGEMENT_ACCOUNT_ID;
+  const testAccountApiKey = config.CASE_MANAGEMENT_APIKEY;
+
+  // Test payload
+  let caseNumber;
+  const commentValue = 'Test comment';
+  const offeringPayload = {
+    name: 'Cloud Object Storage',
+    type: {
+      group: 'crn_service_name',
+      key: 'cloud-object-storage',
+    },
+  };
+
+  const resourceCrn = 'crn:v1:staging:public:cloud-object-storage:global:a/19c52e57800c4d8bb9aefc66b3e49755:61848e72-6ba6-415e-84e2-91f3915e194d::';
+
+  const watchlistPayload = [
+    {
+      realm: 'IBMid',
+      user_id: 'ashwini.pc@ibm.com',
```

Comment:
I suggest we avoid test-account specific values for two reasons. Our tests will break if the test account gets modified. And our tests won't pass if another test account is used.

Reply:
This is a hard issue. Business logic requires that only users in the account to be added to the watchlist.

Although account user modification happens, it doesn't happen frequently.  I've seen account specific details being used in newrelic alerts as well.

I would say that we delegate this account as a test account and no one should make any modification to its without careful consideration of all aspects.